keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -332,21 +414,30 @@ Status.ContainerStatuses[i].Resources to match the desired limit values.
   determine if in-place resize is possible.
 * Impact of memory-backed emptyDir volumes: If memory-backed emptyDir is in
   use, Kubelet will clear out any files in emptyDir upon Container restart.
```

Comment:
Could you please remove this point? We can't touch emptyDir files on container restart.

Reply:
Will do